tests/integration/kafka/src/test/java/io/helidon/messaging/connectors/kafka/KafkaSeTest.java

```diff
@@ -334,4 +349,107 @@ void kafkaHeaderTest() throws InterruptedException {
             messaging.stop();
         }
     }
+
+    @Test
+    void someEventsNoAckWithOnePartition() {
+        LOGGER.fine(() -> "==========> test someEventsNoAckWithOnePartition()");
+        final String GROUP = "group_1";
+        final String TOPIC = TEST_SE_TOPIC_6;
+        Channel<String> fromKafka = Channel.<String>builder()
+                .name("from-kafka")
+                .publisherConfig(KafkaConnector.configBuilder()
+                        .bootstrapServers(KAFKA_SERVER)
+                        .groupId(GROUP)
+                        .topic(TOPIC)
+                        .autoOffsetReset(KafkaConfigBuilder.AutoOffsetReset.EARLIEST)
+                        .enableAutoCommit(false)
+                        .keyDeserializer(LongDeserializer.class)
+                        .valueDeserializer(StringDeserializer.class)
+                        .build()
+                )
+                .build();
+        List<String> uncommit = new ArrayList<>();
+        Channel6 kafkaConsumingBean = new Channel6();
+        Map<String, Object> config = new HashMap<>();
+        config.put("bootstrap.servers", KAFKA_SERVER);
+        config.put("key.serializer", LongSerializer.class.getName());
+        config.put("value.serializer", StringSerializer.class.getName());
```

Comment:
This config seems not to be used anywhere

Reply:
I forgot to remove it. Now that config is in AbstractKafkaTest.produceAndCheck